dotcom-rendering/src/components/Analytics.amp.tsx

```diff
@@ -88,7 +89,7 @@ export const Analytics = ({
 		`<amp-analytics data-block-on-consent="_till_accepted" config="https://uk-script.dotmetrics.net/AmpConfig.json?dom=www.theguardian.com&tag=${ipsosSectionName}">
             <script type="application/json">
                 {
-                    "enabled": "$EQUALS(\${ampGeo(ISOCountry)}, gb)"
+					"enabled": "$IF($EQUALS(\${ampGeo(ISOCountry)}, gb),'true', '$EQUALS(\${ampGeo(ISOCountry)}, aus)')"
```

Comment:
Does it check here if ampGeo is enabled for gb then it enables the ampGeo for aus as well?

Reply:
Yes, that's correct 👍 . The first part of the $IF checks if ampGeo(ISOCountry) is 'gb'. If it is, it returns 'true'. If not, it evaluates the second condition: whether ampGeo(ISOCountry) is 'aus', and returns true or false accordingly. It's essentially a nested IF statement, compacted into a single line—kind of like retro MS Excel formula logic 😄.